simulator/src/dgitx/backend/Backend.kt

```diff
@@ -0,0 +1,120 @@
+package dgitx.backend
+
+import dgitx.BNode
+import git.Git
+import git.RefTxHook
+import git.TxStatus
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.Channel
+import paxos.PxAcceptor
+import paxos.State
+import transaction.*
+import wtf.g4s8.examples.spaxos.*
+import java.util.concurrent.ConcurrentHashMap
+
+class Backend(
```

Comment:
@stepanvalyavskiy does this represent a node instance? please add javadocs to all type definitions